imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -26,13 +27,15 @@
     (1, 128),  # lo-nsw-priority
     (1, 128),  # lo-sw-species
     (1, 128),  # lo-nsw-species
+    (1, 128),  # lo-pha
 ]
 EXPECTED_ARRAY_SIZES = [
     123,  # hskp
     1,  # hi-counters-aggregated
     3,  # hi-counters-singles
     8,  # hi-omni
     4,  # hi-sectored
+    0,  # hi-pha
```

Comment:
Are these expected to be zero length / no data?

Reply:
For now, yes. I need to confirm with Joey how to structure the event data in the CDF (I made sure to put a `TODO` in the code about this). This will be updated in a future PR, but for now it is kind of a placeholder.